src/System.Numerics.Vectors/src/System/Numerics/Vector.cs

```diff
@@ -11,7 +11,6 @@
 using System.Runtime.InteropServices;
 using System.Text;
 
-[assembly: CLSCompliant(false)]
```

Comment:
Looks like the System.Numerics.Vector assembly shouldn't be CLSCompliant.

Reply:
From the docs:

> All assemblies explicitly indicate CLS compliance with CLSCompliantAttribute. If the attribute is not present on an assembly, the assembly is not compliant.

Reply:
Good catch. Need to explicitly set it to false in the csproj.

Those docs seem to apply to regular types of csprojs. We're changing the default behavior with our msbuild conditions in this repo. We set it to true for all except when we explicitly say it should be false for a particular src csproj.

Reply:
Apparently System.Numerics.Vectors had [CLSCompliant(false)] in a bunch of APIs. I'll have to remove them all since the whole assembly is expected to be non-compliant (otherwise there's a build error).

Reply:
Hmm I'm a bit confused now. Maybe @ericstj can double check here.

Reply:
Isn't `[assembly: CLSCompliant(false)]` in a cs file equivalent to `<CLSCompliant>false</CLSCompliant>` in the csproj?

Reply:
S.N.V has `CLSCompliant(true)` at the assembly level to set the default, then it annotated specific methods with `CLSCompliant(false)`

You can see that in the metadata from the 4.5.0 package and in the source: https://github.com/dotnet/corefx/blob/v2.1.30/src/Common/src/CoreLib/System/Numerics/Vector.cs

I'm not sure where this assembly annotation came from but it wasn't there before.

Reply:
Here's a sample that should show you how it works. https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBLANmgExAGoAfAAQCYBGAWACgBtAQwGdWYBbYXATxADCAGQDKAiJwAOubMwB2GABQYoAVxgBKALoNyAZgAEVAwIMBvBgasHGwsROmyFigGbNcHbVYD03g5E5OGAUDDAALbFYDCFUMUIgDDhgDACIAd2YoOWw5AHMTET0ABiK9EAMAFV5JZIgXAwByAQA6AFkGg0iDOQg4uzgAx3kMFMtrfQNVHLjWgwBeAD4DFEoAbjGrDaNDaYMASXmluBX1+gBfIA

Reply:
Weird, I also don't see it in the latest commit of the release/2.1 tag: https://github.com/ericstj/corefx/blob/release/2.1/src/Common/src/CoreLib/System/Numerics/Vector.cs

And I cannot see a particular comment or commit that manually added that line in the migration PR, so I have no idea how it showed up there: https://github.com/dotnet/maintenance-packages/pull/47/commits/44b4394e9ff46d8cf4e5d344c96373ddc90380a6#diff-d22aaff58f9fb931b37fd8130881a13f5084641baa61a012399e76094410e992R14 

I'll fix it now.